File: onlinejudge/baekjoon/Silver/4948 베르트랑 공준.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        while (true) {
            int n = scan.nextInt();
            if (n == 0) {
                break;
            }
            int count = 0;
            for (int i = n + 1; i < 2 * n + 1; i++) {
                if (isPrime(i)) {
                    count += 1;
                }
            }
            System.out.println(count);
        }
    }

    public static boolean isPrime(int n) {
        if (n == 1) {
            return false;
        }
        for (int i = 2; i < (int) (Math.sqrt(n)) + 1; i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }

}

File: onlinejudge/baekjoon/Silver/4948 베르트랑 공준.py
import math

def is_prime(x: int):
    if x == 1:
        return False
    for i in range(2, int(math.sqrt(x)) + 1):
        if x % i == 0:
            return False
    return True

while True:
    n = int(input())
    if n == 0:
        break
    count = 0
    for i in range(n + 1, 2 * n + 1):
        if is_prime(i):
            count += 1
    print(count)
